TransbankDevelopers/transbank-plugin-woocommerce-webpay
# feat/auto-update

Change name plugin and remove old version

The current name generates bureaucratic problems with wordpress.org,
so it cannot be used to upload the plugin to their platform. To make the
change, it is necessary to deactivate and eliminate the previous
versions of the plugin, in this way the commerce will be able to
automatically see and update future updates.

## Test
- Before changing branches, run `./package` and install the previous version of the plugin.
- In `settings: change the data of the `Código de Comercio` and change `Ambiente` to production (To see the effects of the change).
- Change the branch and run `./package` again.
- Install this new version, this must be automatically uninstall the previous version.
- In settings: validate that the values you modified persist.
